build/Makefile

```diff
@@ -391,8 +396,8 @@ do-release:
 	@echo "Now go make the $(RELEASE_VERSION) release on Github!"
 
 setup-test-cluster: $(ensure-build-image)
-	docker run --rm -it $(common_mounts) $(DOCKER_RUN_ARGS) $(build_tag) kubectl apply -f $(mount_path)/build/helm.yaml
-	docker run --rm $(common_mounts) $(DOCKER_RUN_ARGS) $(build_tag) helm init --service-account helm
+	$(DOCKER_RUN) kubectl apply -f $(mount_path)/build/helm.yaml
+	$(DOCKER_RUN) helm init --wait --service-account helm
```

Comment:
Added this `--wait` as it's better when doing combo like kind/minikube-test-cluster && kind/minikube-install it won't failed with an error saying `couldn't find a tiller pod ready`